### DISCUSSION TITLE
Не работает сервер!

### DISCUSSION THREAD

**Discussion by antiknaher** [vvip-68/LedPanelWiFi#25] [Вопросы и ответы]
Здравствуйте. Нужна помощь. Это только у меня или у всех? Не работает сервер?
![Screenshot_2023-11-28-16-53-35-722_com android chrome](https://github.com/vvip-68/LedPanelWiFi/assets/152288192/7058afcb-f2d8-42fe-be34-58ee8095ec52)

**Comment by vvip-68** [comment]
У всех,  кто не читает инструкции или хотя бы FAQ в wiki
